examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,236 @@
+#####################################################################################
+# The MIT License (MIT)
+#
+# Copyright (c) 2015-2024 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#####################################################################################
+# Inference with ONNX Runtime
+import onnxruntime
+import time
+from torchvision import models, transforms as T
+import torch
+from PIL import Image
+import numpy as np
+import argparse
+import os
+import subprocess
+
+#Use most upto date weights
+inception_v3 = models.inception_v3(weights=models.Inception_V3_Weights.DEFAULT, progress=True)
+
+# Download ImageNet labels
+#!curl -o imagenet_classes.txt https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt
+
+def parse_input_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--fp16",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Perform fp16 quantization on the model before running inference',
+    )
+
+    parser.add_argument(
+        "--image_dir",
+        required=False,
+        default="../resnet50/images",
```

Comment:
Is this default value value valid when running this example standalone ?

Reply:
I figured I'd share images between the two so a user can compare

Reply:
we need to change the directory structure in this case. I suggest making a `img_samples` directory that can be shared between the examples.

Reply:
Will do

Reply:
Done